src/Data/Binary/Class.hs

```diff
@@ -239,6 +250,18 @@ instance Binary Integer where
                     let v = roll bytes
                     return $! if sign == (1 :: Word8) then v else - v
 
+-- | /Since: 0.7.x.y/
+#ifdef HAS_FIXED_CONSTRUCTOR
```

Comment:
@phadej Will Data.Fixed have different serialisation formats depending on the branch of this `ifdef`?